deployer/templates/es.yaml

```diff
@@ -41,6 +41,7 @@ objects:
             component: ${ES_CLUSTER_NAME}
             deployment: ${ES_DEPLOYMENT_NAME}
         spec:
+          terminationGracePeriodSeconds: 1800
```

Comment:
Should we be pulling these in via parameter and/or configmap? to allow them to be tweaked in production.  I realize you can edit but then the settings would disappear on re-install